tinygrad/tensor.py

```diff
@@ -204,7 +204,7 @@ def leakyrelu(self, neg_slope=0.01):
     return self.relu() + (-neg_slope*self).relu()
```

Comment:
I don't think this is right.

Assume self = -1
(-0.1*self).relu() = 0.1

You want 
```
self.relu() + (-neg_slope*self).relu()*(-1.0)
```

And abs is
```
self.relu() + (-1.0*self).relu()
```

Please improve the tests for this

Reply:
self.relu - (-neg_slope*self).relu()